packages/color-modes/src/index.tsx

```diff
@@ -155,6 +156,7 @@ export const ColorModeProvider: React.FC = ({ children }) => {
 const noflash = `(function() { try {
   var mode = localStorage.getItem('theme-ui-color-mode');
   if (!mode) return
+  document.documentElement.classList.add('theme-ui-' + mode);
```

Comment:
Why do we add the class both to the `html` and to the `body`?

Reply:
Yeah it should match what is set with the flag.
I wasn't sure if it would be a good idea to add a prop in the `noflash` function.

Reply:
We have the theme in scope, so we could possibly read that option and add only the one line that's needed to the string.